From f76ca78c509d36a55dd0c1fdaa6e947b1e22ebd0 Mon Sep 17 00:00:00 2001
From: VitaminLLLL <liujinchen1118@gmail.com>
Date: Sat, 8 Jun 2024 22:13:40 +0200
Subject: [PATCH] Include only 'misc-modules' and 'scull'

---
 Makefile | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b08c49..c0f2d6b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,6 @@
 
-SUBDIRS =  misc-progs misc-modules \
-           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-	   pci usb\
-	   sbull snull short
+SUBDIRS =  misc-modules \
+           scull
 
 all: subdirs
 
